domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/InformasjonSomRevurderes.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.su.se.bakover.domain.revurdering
+
+import com.fasterxml.jackson.annotation.JsonCreator
+
+data class InformasjonSomRevurderes @JsonCreator(mode = JsonCreator.Mode.DELEGATING) constructor(
+    val informasjonSomRevurderes: Map<Revurderingsteg, Vurderingstatus>,
+) : Map<Revurderingsteg, Vurderingstatus> by informasjonSomRevurderes {
+
+    fun vurdert(revurderingsteg: Revurderingsteg): InformasjonSomRevurderes {
+        return copy(
```

Comment:
Kan man ikke: return copy(informasjonSomRevurderes + revurderingsteg to Vurderingstatus.Vurdert)

Ref: https://kotlinlang.org/docs/map-operations.html#plus-and-minus-operators